sdk/playwrighttesting/Azure.Developer.MicrosoftPlaywrightTesting.TestLogger/src/PlaywrightService.cs

```diff
@@ -187,25 +191,37 @@ private void InitializePlaywrightServiceEnvironmentVariables(string? os = null,
         }
         if (!string.IsNullOrEmpty(os))
         {
-            Environment.SetEnvironmentVariable(ServiceEnvironmentVariable.PlaywrightServiceOs, os);
+            Os = os;
```

Comment:
Have you tested it if this change would honor UseCloudHostedBrowsers value from runsettings ?